### PR TITLE
Feature/addcert

### PR DESCRIPTION
**Pull request by Samuel-Hoskin**

### CODE REVIEW

**Review by Samuel-Hoskin — COMMENTED** [review]
Want content review before merging,

File: one_big_thing/learning/views.py [hunk 1]
```diff
@@ -182,6 +183,7 @@ def get(
             template_name = "record-learning.html"
         time_completed = user.get_time_completed()
         completed_obt = time_completed >= settings.REQUIRED_LEARNING_TIME
+        completed_feedback_survey = user.has_completed_post_survey
```
Comment (252afh) [inline one_big_thing/learning/views.py:186]: You don't need to do this assignment

File: one_big_thing/templates/end-post-survey.html [hunk 2]
```diff
@@ -14,4 +14,6 @@ <h1 class="govuk-panel__title">
       You have completed One Big Thing
     </div>
   </div>
+
+  <p class="govuk-body">You can download your <a target="_blank" rel="noopener noreferrer" href="{{static('Certificate-OBT.pdf')}}" class="govuk-link">One Big Thing certificate</a> to celebrate your completion.</p>
```
Comment (252afh) [inline one_big_thing/templates/end-post-survey.html:18]: If it's a download, generally you don't need to do new tab things

File: one_big_thing/templates/record-learning.html [hunk 3]
```diff
@@ -16,11 +16,20 @@ <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title
...
       <p class="govuk-notification-banner__heading">
-        You have completed {{ humanize_timedelta(data.time_completed, 200, "over 200 hours") }} so far.
+        You have completed {{ humanize_timedelta(data.time_completed, 200, "over 200 hours") }} so far. {% if data.completed_obt %} <strong class="govuk-tag govuk-tag--green">
+          One Big Thing Completed
+        </strong>{% endif %}
       </p>
       {% if data.completed_obt %}
-        <p class="govuk-notification-banner__heading">
-          You should provide <a href="{{url('intro-post-survey')}}">feedback</a> about your One Big Thing experience.
+      {% if data.completed_feedback_survey %}
```
Comment (252afh) [inline one_big_thing/templates/record-learning.html:24]: There's a big load of nothing in the `if` here :P
Just reverse the condition to a `not` if you only want the `else`

**Review by 252afh — APPROVED** [review]

**Pull request merged**